osmodel.mak

```diff
@@ -1,5 +1,15 @@
-# This Makefile snippet detects the OS and the architecture MODEL
-# Keep this file in sync between druntime, phobos, and dmd repositories!
+#   osmodel.mak
+#
+# Detects and sets the macros:
+#
+#   OS         = one of {osx,linux,freebsd,openbsd,netbsd,solaris}
+#   MODEL      = one of { 32, 64 }
+#   MODEL_FLAG = one of { -m32, -m64 }
+#
+# Note:
+#   Keep this file in sync between druntime, phobos, and dmd repositories!
```

Comment:
To make things worse, there's [dlang.org](https://github.com/dlang/dlang.org/blob/master/osmodel.mak) and [tools](https://github.com/dlang/tools/blob/master/osmodel.mak) as well.